src/nupic/research/temporal_memory.py

```diff
@@ -206,6 +212,20 @@ def compute(self, activeColumns, learn=True):
                                                self.predictedSegmentDecrement,
                                                prevActiveCells)
 
+
+  def activateDendrites(self, learn=True):
```

Comment:
ha, you scared me. I think it's in the same order as before? It calls `activateCells` before `activateDendrites`.